workflow/rules/ref.smk

```diff
@@ -0,0 +1,69 @@
+rule get_genome:
+    output:
+        "resources/ref/genome.fasta"
+    log:
+        "logs/ref/get-genome.log"
+    params:
+        species=config["resources"]["ref"]["species"],
+        datatype="dna",
+        build=config["resources"]["ref"]["build"],
+        release=config["resources"]["ref"]["release"]
+    cache: True
+    wrapper:
+        "0.64.0/bio/reference/ensembl-sequence"
+
+rule get_annotation:
+    output:
+        "resources/ref/annotation.gtf"
+    params:
+        species=config["resources"]["ref"]["species"],
+        release=config["resources"]["ref"]["release"],
+        build=config["resources"]["ref"]["build"],
+        fmt="gtf",
+        flavor=""
+    log:
+        "logs/ref/get_annotation.log"
+    cache: True  # save space and time with between workflow caching (see docs)
+    wrapper:
+        "0.64.0/bio/reference/ensembl-annotation"
+
+rule gtf2bed:
+    input:
+        "resources/ref/annotation.gtf"
+    output:
+        "resources/ref/genome.bed"
+    conda:
+        "../envs/perl.yaml"
+    shell:
+        "../workflow/scripts/gtf2bed {input} > {output}"
```

Comment:
We could add a log file that captures stderr, here (assuming there is a `log:` entry above):
```suggestion
        "../workflow/scripts/gtf2bed {input} > {output} 2> {log}"
```